examples/test_cheque.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+
+from pylatex import Document, MiniPage, TextBlock, MultiColumn, Tabu
+from pylatex.utils import huge, bold, header2, small1, horizontal_skip, \
+    horizontal_fill, vertical_skip, page_break
+
+doc = Document(page_numbers=False)
```

Comment:
Could you use the `with` syntax in your examples? That makes them much more readable.
Check out the other examples to see what I mean, for instance this one:
https://jeltef.github.io/PyLaTeX/latest/examples/subfigure.html